src/earn/EarnEnterAmount.tsx

```diff
@@ -683,6 +797,29 @@ const styles = StyleSheet.create({
     flexWrap: 'wrap',
     textAlign: 'left',
   },
+  gap8: {
```

Comment:
nit: would be good to keep all `txDetails` styles together

Reply:
Fixed in afa31bfc56efd1ba480e2d8bf8e378f2a353f02b!